Compile warnings for visual expression node

**Godot version:**
https://github.com/godotengine/godot/commit/7d42ca838486ce4a737e074b60fffd81de1bc971

**OS/device including version:**
* Mac
* Not a graphics bug.

**Issue description:**

@Chaosus 

Code is c++11.

```
[ 77%] editor/plugins/visual_shader_editor_plugin.cpp:987:38: warning: use of enumeration in a nested name specifier is a C++11 extension [-Wc++11-extensions]
                box_size.x -= text_box->get_margin(Margin::MARGIN_LEFT);
                                                   ^
editor/plugins/visual_shader_editor_plugin.cpp:989:38: warning: use of enumeration in a nested name specifier is a C++11 extension [-Wc++11-extensions]
                box_size.y -= text_box->get_margin(Margin::MARGIN_TOP);
                                                   ^
```

**Steps to reproduce:**
Compile.

--- a/editor/plugins/visual_shader_editor_plugin.cpp
+++ b/editor/plugins/visual_shader_editor_plugin.cpp
@@ -984,9 +984,9 @@ void VisualShaderEditor::_set_node_size(int p_type, int p_node, const Vector2 &p
 				box_size.x = gn->get_size().x;
 			}
 		}
-		box_size.x -= text_box->get_margin(Margin::MARGIN_LEFT);
+		box_size.x -= text_box->get_margin(MARGIN_LEFT);
 		box_size.x -= 28 * EDSCALE;
-		box_size.y -= text_box->get_margin(Margin::MARGIN_TOP);
+		box_size.y -= text_box->get_margin(MARGIN_TOP);
 		box_size.y -= 28 * EDSCALE;
 		text_box->set_custom_minimum_size(Size2(box_size.x, box_size.y));
 		text_box->set_size(Size2(1, 1));

--- a/scene/resources/visual_shader.cpp
+++ b/scene/resources/visual_shader.cpp
@@ -2204,6 +2204,7 @@ String VisualShaderNodeExpression::generate_code(Shader::Mode p_mode, VisualShad
 	if (post_symbols.empty()) {
 		post_symbols.push_back("\t");
 		post_symbols.push_back("\n");
+		post_symbols.push_back(";");
 		post_symbols.push_back("}");
 		post_symbols.push_back("]");
 		post_symbols.push_back(")");

